ros_ign_gazebo/CMakeLists.txt

```
@@ -14,57 +14,63 @@ find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(std_msgs REQUIRED)
 
+# TODO(CH3): Deprecated. Remove on tock.
+if("$ENV{GZ_VERSION}" STREQUAL "" AND NOT "$ENV{IGNITION_VERSION}" STREQUAL "")
+  message(DEPRECATION "Environment variable [IGNITION_VERSION] is deprecated. Use [GZ_VERSION] instead.")
+  set(ENV{GZ_VERSION} ENV{IGNITION_VERSION})
```

This wasn't working for me, it works like this:

```suggestion
  set(ENV{GZ_VERSION} $ENV{IGNITION_VERSION})
```